test/functional/ui/screen_basic_spec.lua

```diff
@@ -1,13 +1,21 @@
 local helpers = require('test.functional.helpers')
 local Screen = require('test.functional.ui.screen')
-local clear, feed, execute = helpers.clear, helpers.feed, helpers.execute
+local spawn, set_session, feed, execute = helpers.spawn, helpers.set_session, helpers.feed, helpers.execute
 local insert, wait = helpers.insert, helpers.wait
 
 describe('Screen', function()
   local screen
+  local nvim_prog = os.getenv('NVIM_PROG') or 'build/bin/nvim'
+  local nvim_argv = {nvim_prog, '-u', 'NONE', '-i', 'NONE', '-N',
+                   '--cmd', 'set shortmess+=I background=light noswapfile noautoindent',
+                   '--embed'}
 
   before_each(function()
-    clear()
+    if session then
+      session:exit(0)
+    end
+    local screen_nvim = spawn(nvim_argv)
+    set_session(screen_nvim)
```

Comment:
I'm confused about why this as needed: it initializes a session with laststatus=2, yet none of the tests were changed. Is it still needed?